compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/test/invalid.mlir

```diff
@@ -0,0 +1,101 @@
+// RUN: iree-opt --split-input-file --verify-diagnostics %s
+
```

Comment:
I tried to add the following test to verify dynamic shape failure. The error seems to be returned, but somehow the test still fails. Couldn't figure out what I am doing wrong immediately, so if anyone has an idea?
```
// expected-error @+1 {{should encapsulate static memref}}
func.func @test(!amdaie.logicalobjectfifo<memref<?x8x16xi32, 1>>)
```

Reply:
Have you figured this out? I didn't see build failure.

Reply:
Have you figured this out? I didn't see

Reply:
No, I didn't put it into the file as it doesn't pass for me locally. However, it reports the error that is expected..

Reply:
can you add the FileCheck, we can help if we see the failure in CI.

Reply:
Added it, waiting for a runner.. Could it be offline as I don't see anything else running?

Reply:
I believe I figured it out :) see https://github.com/nod-ai/iree-amd-aie/pull/267/commits/5aaea5c0a0cdfd1f36e21b3bae8067eefaeaadc4

Reply:
Yeah, CI passed

Reply:
there is no FileCheck here?

Reply:
oh I see, thats how the invalid tests work look likes.